libs/expo/betterangels/src/lib/screens/AddEditClient/HouseholdMembers/Household/index.tsx

```diff
@@ -0,0 +1,206 @@
+  const { control, setValue, watch } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
+
+  const relationship = watch(
+    `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`
+  );
+
+  const householdMembers = watch('householdMembers') || [];
+
+  const handleRemove = () => {
+    remove(index);
+  };
+
+  const handleReset = () => {
+    setValue(
+      `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].name` as `householdMembers.${number}.name`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].gender` as `householdMembers.${number}.gender`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].dateOfBirth` as `householdMembers.${number}.dateOfBirth`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`,
+      null
+    );
+  };
+
+  if (!relationship) {
+    return (
+      <Select
+        boldLabel
+        labelMarginLeft="xs"
+        label="Type of Relationship"
+        placeholder="Select Relationship"
+        defaultValue={(relationship as RelationshipTypeEnum | undefined) ?? ''}
+        onValueChange={(enumValue) =>
+          setValue(
+            `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`,
+            enumValue as RelationshipTypeEnum
+          )
+        }
+        items={Object.entries(enumDisplayRelevant)
+          .filter(
+            ([enumValue]) =>
+              enumValue !== 'CURRENT_CASE_MANAGER' &&
+              enumValue !== 'PAST_CASE_MANAGER' &&
+              enumValue !== 'ORGANIZATION'
+          )
+          .map(([enumValue, displayValue]) => ({
+            displayValue: displayValue,
+            value: enumValue,
+          }))}
+      />
+    );
+  }
+
+  return (
+    <View
+      style={{
+        gap: Spacings.sm,
+        borderRadius: Radiuses.xs,
+        borderColor: Colors.NEUTRAL_LIGHT,
+        borderWidth: 1,
+        backgroundColor: Colors.WHITE,
+        paddingTop: Spacings.md,
+        paddingBottom: Spacings.lg,
+        paddingHorizontal: Spacings.sm,
+      }}
+    >
+      <TextBold size="lg">{enumDisplayRelevant[relationship]}</TextBold>
+      {householdMembers[index].relationshipToClient ===
+        RelationshipTypeEnum.Other && (
+        <Input
+          placeholder="Enter type of relationship"
+          label="Type of Relationship"
+          name={`householdMembers[${index}].relationshipToClientOther`}
+          control={control}
+        />
+      )}
+      <Input
+        placeholder="Name"
+        autoCorrect={false}
+        label="Name"
+        name={`householdMembers[${index}].name`}
+        control={control}
+      />
+
+      <View style={{ gap: Spacings.xs }}>
```

Comment:
can we create a shared `Gender` component for this and `PersonalInfo`?

Reply:
@shiozakiyuko do you think gender would be better as a dropdown? the full list takes up almost half of the screen
<img width="312" alt="image" src="https://github.com/user-attachments/assets/3c4525b0-5868-4ffc-831e-93e132b45492">

Reply:
done